Entity/SingleTypeIndex.php

```diff
@@ -205,4 +207,20 @@ public function getContentType()
     {
         return $this->contentType;
     }
+
+    /**
+     * Specify data which should be serialized to JSON
+     * @link https://php.net/manual/en/jsonserializable.jsonserialize.php
+     * @return mixed data which can be serialized by <b>json_encode</b>,
+     * which is a value of any type other than a resource.
+     * @since 5.4.0
+     */
+    public function jsonSerialize()
+    {
+        $json = new JsonClass(get_object_vars($this), __CLASS__);
+        $json->removeProperty('id');
+        $json->removeProperty('environment');
+
```

Comment:
So, what the meaning of breaking this link? Is it not better to call the reconnect command?

Reply:
When exporting and importing json files between projects, we cannot guarantee that the same environment names are used in both projects. This is also done for the contenttype json.

When importing, the environment to use should be specified (as we allready require for content types)

Reply:
We cannot

> I'm not comfortable with the links between entities

Do you mean the "extends JsonDeserializable" ? Why are you not comfortable with that link?